packages/loot-design/src/components/autocomplete-styles.js

```diff
@@ -0,0 +1,51 @@
+import { colors } from '../style';
+
+const colourStyles = {
+  control: styles => ({
+    ...styles,
+    backgroundColor: 'white',
+    border: '1px solid rgb(208, 208, 208)',
+    borderRadius: 4,
+    outline: 0,
+    marginLeft: -1,
+    marginRight: 1,
+    padding: '5px 3px',
+    minHeight: 'auto',
+  }),
+  input: styles => ({
+    ...styles,
+    padding: 0,
+    margin: 0,
+  }),
+  menu: (styles, { selectProps }) => ({
+    ...styles,
+    backgroundColor: colors.n1,
+    marginTop: 2,
+    zIndex: 5000,
+    position: selectProps.embedded ? 'relative' : styles.position,
```

Comment:
This will fix the menu items poking out of the corners of the menu:

```suggestion
    position: selectProps.embedded ? 'relative' : styles.position,
    overflow: hidden;
```